static/app/components/lastCommit.tsx

```diff
@@ -41,6 +41,11 @@ function LastCommit({commit, className}: Props) {
       }
       return `${truncated}\u2026`;
     }
+
+    if (commit.repository !== undefined) {
+      const commitURL = commit.repository.url + '/commit/' + commit.id;
```

Comment:
this could be an issue in the case the repository website formats their urls a different way, but atm, there is no way to retrieve the commit url from `commit`